recipes/pymc/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pymc" %}
+{% set version = "2.3.6" %}
+{% set sha256 = "e287a7d1821d55a41079fe0289d70b6f2ea905f4c6b1a9207bdf35425bdcae37" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
`pymc` does not seem to use `setuptools`, so you will need to remove the flags `--single-version-externally-managed --record record.txt` and the dependency below.